src/main/java/uk/gov/hmcts/reform/bulkscanprocessor/tasks/processor/ZipVerifiers.java

```diff
@@ -114,7 +115,7 @@ static ZipInputStream sha256WithRsaVerification(ZipStreamWithSignature zipWithSi
 
     static void verifyFileNames(Set<String> fileNames) {
         if (!(fileNames.size() == 2 && fileNames.containsAll(asList(DOCUMENTS_ZIP, SIGNATURE_SIG)))) {
-            throw new DocSignatureFailureException(
+            throw new InvalidZipFilesException(
```

Comment:
does it need to be caught specifically somewhere similarly as `DocSignatureFailureException`?

Reply:
good point, I'll check